### PR TITLE
removed unwanted  css

### PR DESCRIPTION
**Pull request by Ansonsd5**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -5,58 +5,51 @@
...
 
-  --button-fw : 600;
+  --button-fw: 600;
 }
 
-.btn {
-  background-color: var(--primary-color);
-  color: var(--text-color);
+.btn,
+.btn-outline {
```
Comment (jsuryakt) [inline index.css]: We don't need .btn-outline here
.btn-outline should only define border for the button

Comment (jsuryakt) [reply index.css]: Rest of the styles should be similar, btn will not have any border by default
But btn-outline will define a border

File: index.css [hunk 2]
```diff
@@ -5,58 +5,51 @@
...
-    background-color: var(--active-color);
-    color: var(--text-color)
+.btn-outline {
+  background-color: var(--secondary-color);
+  border: 1px solid var(--primary-color);
 }
 
-.btn:hover{
- background-color: var(--secondary-color);
+button[class*="btn"]:active {
```
Comment (jsuryakt) [inline index.css]: Then this should work with .btn
Anyways while using a outlined button user must use it in this way
`<button class="btn btn-outlined">Button</button>`

So btn will be there

File: index.css [hunk 3]
```diff
@@ -5,58 +5,51 @@
...
-    transition: all 0.2s ease-in-out;
+.btn {
+  background-color: var(--primary-color);
+  border: none;
 }
 
-button[class*='btn']:active{
-    background-color: var(--active-color);
-    color: var(--text-color)
+.btn-outline {
```
Comment (jsuryakt) [inline index.css]: We don't need bg changed with btn-outline doesn't define it

Comment (jsuryakt) [reply index.css]: We can have different classes for colors if reqd

```
<button class="btn btn-secondary">Button</button>
<button class="btn btn-primary">Button</button>
<button class="btn btn-info">Button</button>
<button class="btn btn-warning">Button</button>
```

```
example

.btn-info {
background: yellow;
}
```
etc

File: index.css [hunk 4]
```diff
@@ -5,58 +5,51 @@
...
-button[class*='btn']:disabled{
-    background: var(--secondary-color);
+.btn-outline:hover {
+  background-color: var(--primary-color);
 }
 
-button[class*='btn']:disabled{
-    opacity: 0.4;
-    cursor: default;
+button[class*="btn"]:disabled {
```
Comment (jsuryakt) [inline index.css]: .btn

File: index.html [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
-    </form>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <button class="btn">Button</button>
+    <button class="btn" disabled>Button</button>
+    <hr />
+    <button class="btn-outline">Button</button>
```
Comment (jsuryakt) [inline index.html]: Re-use the btn class here
`<button class="btn btn-outline">Button</button>`

**Review by jsuryakt — CHANGES_REQUESTED** [review]

File: index.css (hunk 4 above)
Comment (Ansonsd5) [reply index.css]: ok

File: index.css [hunk 6]
```diff
@@ -5,58 +5,56 @@
...
   --error-color: #b50001;
   --focus-color: #00d0e480;
-  --shadow-color: rgba(33 51 67 0.12);
+  --shadow-color: #2133431f;
   --white-color: #ffffff;
-  --black-color: rgba(255, 255, 255, 0);
-  --active-color: #d44325;
+  --black-color: #ffffff00;
+  --active-color: #d44326;
+  --transperent-color: #ffffff00;
```
Comment (jsuryakt) [inline index.css]: typo transparent

File: index.css [hunk 7]
```diff
@@ -5,58 +5,58 @@
...
-    border-radius: 5px;
-    padding: 0.5rem 1.5rem;
-    cursor: pointer;
-    line-height: 1.2;
-    font-weight:var(--button-fw)
-    transition: all 0.2s ease-in-out;
+
+
+.btn:active,
+.btn-outline:active {
```
Comment (jsuryakt) [inline index.css]: .btn-outline not needed here

File: index.css [hunk 8]
```diff
@@ -5,58 +5,58 @@
...
+  background: var(--active-color) !important;
+}
+.btn {
```
Comment (jsuryakt) [inline index.css]: Can add it along with all the .btn styles

File: index.css [hunk 9]
```diff
@@ -5,58 +5,58 @@
...
-.btn-outline:hover{
-background-color: var(--primary-color);
+.btn-outline:hover {
+  background: var(--primary-color);
 }
 
-button[class*='btn']:disabled{
-    background: var(--secondary-color);
+.btn:disabled,
+.btn-outline:disabled {
```
Comment (jsuryakt) [inline index.css]: can remove btn-outline here

**Review by jsuryakt — APPROVED** [review]
LGTM

**Pull request merged**